### PR TITLE
runtime: Set post-migration BaseCallFilter

### PR DESCRIPTION
**Pull request by NunoAlexandre**
Closes https://github.com/centrifuge/centrifuge-chain/issues/623

_Waiting on the CI build as I cannot build on my M1 machine_

### CODE REVIEW

**Comment by NunoAlexandre** [comment]
@mikiquantum maybe you can have a look at this already. I would like to add some tests verifying this works as expected still.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: build
 on:
   pull_request:
   push:
-    branches: [master]
+    branches: [master, nuno/disable-almost-all-calls]
```
Comment (NunoAlexandre) [inline .github/workflows/build.yml:5]: todo: undo this before merging

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -148,8 +148,35 @@ parameter_types! {
...
+			c,
+			// Calls for runtime upgrade
+			Call::System(frame_system::Call::set_code{..}) |
+			Call::System(frame_system::Call::set_code_without_checks{..}) |
+            // Council-related calls
+            Call::Council(..) |
+            // Democracy-related calls
+            Call::Democracy(..) |
+			// Calls that are present in each block
+			Call::Timestamp(pallet_timestamp::Call::set{..})
```
Comment (NunoAlexandre) [inline runtime/src/lib.rs:173]:
```suggestion
			// Calls for runtime upgrade
			Call::System(frame_system::Call::set_code{..}) |
			Call::System(frame_system::Call::set_code_without_checks{..}) |
      // Council-related calls
      Call::Council(..) |
      // Democracy-related calls
      Call::Democracy(..) |
			// Calls that are present in each block
			Call::Timestamp(pallet_timestamp::Call::set{..})
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (NunoAlexandre) [inline .github/workflows/build.yml:5]:
```suggestion
    branches: [master]
```

File: .travis.yml [hunk 3]
```diff
@@ -11,7 +11,7 @@ cache: cargo
 
 branches:
   only:
-  - master
+  - nuno/disable-almost-all-calls
```
Comment (NunoAlexandre) [inline .travis.yml:14]:
```suggestion
  - master
```

File: .travis.yml [hunk 4]
```diff
@@ -38,7 +38,7 @@ jobs:
       name: "Build binary"
       script: TARGET=build-client travis_wait 180 ./ci/script.sh
     - stage: "Build runtime"
-      if: (NOT type IN (pull_request)) AND (branch = master || tag IS present)
+#      if: (NOT type IN (pull_request)) AND (branch = master || tag IS present)
```
Comment (NunoAlexandre) [inline .travis.yml:41]:
```suggestion
      if: (NOT type IN (pull_request)) AND (branch = master || tag IS present)
```

**Review by NunoAlexandre — COMMENTED** [review]
Changes needed before merging

**Comment by mikiquantum** [comment]
@NunoAlexandre looks good! Thank you, will try to figure out how to run the builds.

**Review by mikiquantum — APPROVED** [review]
👍

**Pull request merged**